composer.json

```diff
@@ -169,9 +171,9 @@
         "spryker/documentation-generator-rest-api": "^1.15.0",
         "spryker/dummy-marketplace-payment": "^0.2.5",
```

Comment:
Where is https://github.com/spryker/suite-nonsplit/pull/7471/files#diff-66f8b2f09b822a2e1db3e1f680cffb1a72731c316e698495b5dfe61b6b0033ef integration?

Reply:
For b2c and b2c mp it was already there.
For b2b and b2b mp it must not be because they do not have GiftCard functionality (discussed with @Nataliia-Popkova )